### ISSUE TITLE
Work on newer version.

### ISSUE THREAD

**Issue by aroberge** [aroberge/ideas#22]
Goals for new version:

- [ ] Support for IPython/Jupyter
- [ ] Remove argument passing to transformers for showing original or transformed code, and rely on a configuration settings.
- [ ] Revise the documentation to show the command line option as the default, instead of importing, adding hook, etc.
- [ ] Add an example showing how to load a custom transformer from the command line.
- [ ] Add info about adding `*kwargs` to future-proofing transformers.
- [ ] Separate out encodings example, reducing the emphasis.
- [ ] See if it is possible to adapt/update the failing examples for Python 3.8+
- [ ] See if it is possible to show the transformed code for Python with AST for Python 3.9 as it includes `unparse` by default.

**Comment by tomkcook** [comment]
Is this why simple examples fail on Python 3.10?  This for instance:

```transforms.py
import ideas

def my_transform(source, **_kwargs):
    return source

def add_hook(**_kwargs):
    hook = ideas.import_hook.create_hook(transform_source=my_transform)
    return hook
```

```test.py
my_var = "test value"
```

```
$ python3 -m ideas test -a transforms
Module <module 'transforms' from '/home/tkcook/git/idea/transforms.py'> does not contain a function named add_hook
```

**Comment by tomkcook** [comment]
Relevant versions:

```
$ venv/bin/pip freeze
ideas==0.1.3
token-utils==0.1.8
$ python3 --version
Python 3.10.6
```

**Comment by tomkcook** [comment]
No, never mind, that's because an `AttributeError` in your `add_hook` function has the same effect as a missing `add_hook` function.

**Comment by aroberge** [comment]
Try to upload the newest version (0.1.4) - just uploaded to pypi.
Also, instead of importing ideas and trying to call `ideas.import_hook...`, do
```
from ideas import import_hook
```
